package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is this in the right location? I think you mean to do the `yarn add` within the `frontend` dir.

Reply:
doh!

Reply:
these are fixed